### ISSUE TITLE
New file format request - AdBlockPlus

### ISSUE THREAD

**Issue by yodaluca23** [Isaaker/Spotify-AdsList#37]
### What is the name of your adblocker?

AdBlockPlus

### What is the name of your adblocker?

AdBlockPlus

### Please, provide the URL to the adblocker official web

https://adblockplus.org

### Please, provide the URL to the adblocker documentation

https://help.adblockplus.org/hc/en-us/articles/360062733293-How-to-write-filters

### Format Example

```shell
https://easylist-downloads.adblockplus.org/v3/full/abp-filters-anti-cv.txt
```

### Extra Info about the request

I use uBlockOrigin on all my devices, but my institution only has ABP allowed...

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct
- [X] I read the avaible Documentation
- [X] I have checked that there is no issue on the same subject
- [X] I have checked that my issue does not contain any relevant security data (Passwords, Secrets or Users)
- [X] I have checked that my issue is not a Security Vulneability. Please, read Security Policy.

**Comment by Isaaker** [comment]
Hi @yodaluca23, thanks for the suggestion. I will try to add this new format.

**Comment by Isaaker** [comment]
Hello, new ABP format included in the latest version 1.1.0.
